packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -178,6 +186,9 @@ export const getServerSideProps = async function getServerSideProps(
       serializableForm.fields
     );
 
+    // TODO: To be done using sentry tracing
+    console.log("Server-Timing", getServerTimingHeader(timeTaken));
```

Comment:
`timeTaken` is already being computed, so I will just use the same approach for now for quick release. I will later move it to Sentry.